functions/Get-DbaAvailabilityGroup.ps1

```diff
@@ -90,7 +90,7 @@ function Get-DbaAvailabilityGroup {
 				if ($IsPrimary) {
 					$defaults = 'ComputerName','InstanceName','SqlInstance','Name as AvailabilityGroup','IsPrimary'
 					$value = $false
-					if ($serverName -eq $ag.PrimaryReplicaServerName) {
+					if ($ag.PrimaryReplicaServerName -eq $serverName) {
```

Comment:
Yeah, @awickham10 edited my initial comment but we should actually be using `$server.Name` as the comparison and not the string passed into the command. Good catch on this one.